contracts/governance/FeeSharingProxy/FeeSharingProxyStorage.sol

```diff
@@ -64,6 +64,13 @@ contract FeeSharingProxyStorage is Ownable {
 	 */
 	uint256 internal reentrancyLock = REENTRANCY_GUARD_FREE;
 
+	/**
+	 * @dev Additional storage for converter whitelist mechanism.
+	 * @dev Initialization here does not works. We need to create a separate setter & getter.
+	 * @dev Just set the visibility to internal should be fine.
+	 */
+	mapping(address => bool) internal isWhitelistedConverter;
```

Comment:
pls use EnumerableBytes32 set - you can skip values validations (existence checks) and get whitelisted addresses list easily

Reply:
You mean EnumerableAddressSet?
It will cost more gas then?
Note: this isWhitelistedConverter is already been deployed to the testnet, the storage collision concern should be ruled out here because isWhitelistedConverter is just a mapping that reserved empty data for this slot.

Reply:
This PR has been merged into the PR #385 
This fix moved to https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/385/commits/dd192177090b2ee1c70689db8519802ff24bfcf9